src/pages/search-results.js

```diff
@@ -0,0 +1,272 @@
+                  key={`searchpage-${pageNumber}`}
+                  disabled={pageNumber === page}
+                  to={`${slug}/?query=${query}&page=${pageNumber}`}
+                >
+                  <PaginationButton
+                    disabled={pageNumber === page}
+                    css={css`
+                      padding: 0.25rem 0.35rem;
+                      ${pageNumber === page &&
+                      css`
+                        background: var(--primary-text-color);
+                        color: var(--primary-background-color);
+                        opacity: 1;
+                      `}
+                    `}
+                  >
+                    {pageNumber}
+                  </PaginationButton>
+                </Link>
+              );
+            })}
+            <Link
+              disabled={!hasNextPage}
+              to={`${slug}/?query=${query}&page=${nextPage}`}
+            >
+              <PaginationButton
+                disabled={!hasNextPage}
+                css={css`
+                  padding: 0.25rem 0.35rem;
+                  margin-left: 0.5rem;
+                `}
+              >
+                Next
+                <Icon
+                  name="fe-arrow-right"
+                  css={css`
+                    margin-left: 0.25rem;
+                  `}
+                />
+              </PaginationButton>
+            </Link>
+          </PaginationContainer>
+        </>
+      )}
+    </PageContainer>
+  );
+};
+
+const PageContainer = styled.div`
+  font-size: 1.125rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  h2 {
+    font-weight: normal;
+    margin-bottom: 1rem;
+  }
+`;
+
+const LoadingContainer = styled.div`
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+const PaginationContainer = styled.div`
+  display: flex;
+  max-width: 760px;
+  justify-content: center;
+  align-items: flex-end;
+  margin: 3rem auto 0;
+  a {
+    margin: 0 0.25rem 0;
+    display: flex;
+    button {
+      &:hover {
+        color: var(--brand-button-primary-accent-hover);
+        border-color: var(--brand-button-primary-accent-hover);
+      }
+    }
+    text-decoration: none;
+    &[disabled] {
+      pointer-events: none;
+      button {
+        border-color: --system-text-muted-light;
+        color: --system-text-muted-light;
+      }
+    }
+  }
+`;
+
+const PaginationButton = ({ children, ...props }) => (
+  <Button {...props} variant={Button.VARIANT.OUTLINE} size={Button.SIZE.SMALL}>
+    {children}
+  </Button>
+);
+
+const Result = ({ result }) => {
+  return (
+    <Surface
+      as={Link}
+      to={result.url}
+      css={css`
+        em {
+          color: #00ac69;
+          font-style: normal;
+        }
+        margin-bottom: 1rem;
+        box-shadow: none;
+        color: var(--primary-font-color);
+        &:hover {
```

Comment:
after trying it out, i don't think hovering over a result that it's obvious enough you can click the entire result. what do you think about adding a `&:hover h3 { text-decoration: underline; }` here?

Reply:
poifect!